plugins/logitech-bulkcontroller/proto/meson.build

```diff
@@ -0,0 +1,21 @@
+
+gen = generator(protoc, \
+  output    : ['@BASENAME@.pb-c.c', '@BASENAME@.pb-c.h'],
+  arguments : ['--proto_path=@CURRENT_SOURCE_DIR@', '--c_out=@BUILD_DIR@', '@INPUT@'])
+
+src = [
+       'antiflicker.proto',
+       'ble_cfg.proto',
+			 'crash_info.proto',
+       'device_attestation.proto',
+       'device_common.proto',
+       'device_info.proto',
+       'device_mode.proto',
+       'device_request.proto',
+       'device_time.proto',
+       'firmware_update.proto',
+			 'rightsight.proto',
+       'ota_manifest.proto',
+       'usb_msg.proto',
+       ]
```

Comment:
nit: whitespace not lined up

Reply:
I don't think we need 80% of them too...

Reply:
As this is probably lifted from the Windows or Mac updater - is there some value in keeping them split up this way for when there needs to be an update some day for a new device?

Reply:
API changes not that frequent.